### PR TITLE
List counseling centers

### PR DESCRIPTION
**Pull request by rebekkaschaub**
- Add endpoint to get counseling centers
- simple test

- list counseling centers in frontend
- basic styling

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/model/CounselingCenter.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class CounselingCenter {
+    private String name;
```
Comment (fabianschmauder) [inline backend/src/main/java/de/neuefische/backend/model/CounselingCenter.java:15]: i would add a unique id field.
A unique id makes it easier to access a specific Center and you can also use the id as key in your frontend.

File: frontend/src/pages/ResultsListPage.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <Wrapper>
+      <Header>
+        <h1>Beratungsstellen in der Nähe</h1>
+      </Header>
+
+      <div>
+        {counselingCenters.map((center) => (
+          <CounselingCenterCard key={center.url} counselingCenter={center} />
```
Comment (fabianschmauder) [inline frontend/src/pages/ResultsListPage.js:25]: would be better to use a real id

**Review by fabianschmauder — COMMENTED** [review]
Looks good. I would add a unique id for your model. 😉🕺🏻

**Review by yasminwaldeck — APPROVED** [review]
Find ich gut :)

File: frontend/src/pages/ResultsListPage.js (hunk 2 above)
Comment (rebekkaschaub) [reply frontend/src/pages/ResultsListPage.js]: done :)

**Pull request merged**